upgrading/automated_upgrades.adoc

```diff
@@ -731,13 +731,13 @@ Linux Atomic Host.
 When upgrading  {product-title}, you must upgrade the set of nodes where
 GlusterFS pods are running.
 
-Special consideration must be taken when upgrading these nodes, as `drain` and
+Use special consideration when upgrading these nodes, as `drain` and
```

Comment:
This phrasing of "Use special consideration" sounds odd to me. Consider rewording, maybe something like "Use caution when..."?

Reply:
@ bergerhoffer. agree. I changed it to "use care"